client/components/product-card/action.jsx

```diff
@@ -3,16 +3,22 @@
  */
 import React from 'react';
 import PropTypes from 'prop-types';
+import { noop } from 'lodash';
 
 /**
  * Internal dependencies
  */
 import Button from 'components/button';
 
-const ProductCardAction = ( { intro, label, onClick } ) => (
+const ProductCardAction = ( { intro, label, onClick, isPrimary, href } ) => (
```

Comment:
I'd totally suggest going for `primary` instead of `isPrimary` - not for any other reason, but to be consistent with the `<Button />` component prop. This is already a good reason.

```suggestion
const ProductCardAction = ( { intro, label, onClick, primary, href } ) => (
```